README.md

```diff
@@ -112,7 +101,12 @@ The type of `formatted` will depend on the formatter function and can be safely
 
 Three formatters are included out of the box:
 
-* `SQLDatabaseFormatter` returns format to SQL technology such as PostgreSQL and MySQL.
+* `SQLDatabase` returns format to SQL technology such as PostgreSQL and MySQL.
```

Comment:
This sentence doesn't quite parse grammatically.  Try "... returns a database URL used by most MySQL and PostgreSQL libraries"?

Reply:
Thank you